README.md

```diff
@@ -334,9 +341,9 @@ Sample response:
 ## Changelog
 
```

Comment:
Could you add 0.10.0 here with a note stating query params are now present in READ?